src/content/cloud/github-actions.mdx

```diff
@@ -14,16 +14,11 @@ GitHub Actions gives you the flexibility to build automated software development
 The GitHub Actions for Okteto Cloud are available directly from [the GitHub Markeplace](https://github.com/marketplace?type=actions&query=okteto).
 
 - [Activate Namespace](https://github.com/marketplace/actions/activate-namespace)
-- [Apply](https://github.com/marketplace/actions/kubectl-apply)
 - [Build](https://github.com/marketplace/actions/okteto-build)
-- [Context](https://github.com/marketplace/actions/okteto-context)
```

Comment:
okteto-context is not deprecated, it wasn't published in the marketplace due to a configuration issue.

Reply:
So, the broken will stay broken till the action is published again. Should it be removed temporary from the documentation? or we just keep it?

Reply:
Let's keep it and we'll fix it. What is exactly broken with the context action? The link?

Reply:
Yes, the link leads to the github actions marketplace page https://github.com/marketplace/actions/okteto-context but this link return 404.

I will undelete the context change I made. But it might be an idea to use the link of the context code repo https://github.com/okteto/context instead of the broken link to the marketplace until the action is published again.